README.md

```diff
@@ -166,11 +170,19 @@ of the notebooks.
 1. If a python execution fails to import a module, make sure to add current
 directory to your PYTHONPATH.
 
-1. If starting the server gives a CONNECTION\_ERROR, make sure MongoDB is
+2. If starting the server gives a CONNECTION\_ERROR, make sure MongoDB is
```

Comment:
For the record, you shouldn't need to do this because markdown should automatically generate numbers. For example, in the list below, all my numbers started with 1. You can try to add a github comment and check for yourself. But it doesn't hurt anything, so I won't hold up the merge for this.

1. Yawen
1. Wang
1. is
1. awesome